pkg/controllers/dnspolicy/dnspolicy_healthchecks.go

```diff
@@ -123,10 +124,17 @@ func (r *DNSPolicyReconciler) expectedHealthCheckProbesForGateway(ctx context.Co
 	}
 
 	for _, address := range gw.Status.Addresses {
-		matches := strings.Split(address.Value, "/")
-		if len(matches) != 2 {
-			log.V(3).Info(fmt.Sprintf("%s cannot be processed. expected <clsutername>/<hostname/ipaddress>", address.Value))
-			return nil
+		addressValue := ""
```

Comment:
currently if it is a hostname then it never worked? Should it work if there is a host instead of an IP?

Reply:
We don't have to fix that here, but if we would expect it to work then lets create an issue and if we don't let make sure we report it to the end user via status